tests/acceptance/steps/util.py

```diff
@@ -147,6 +148,6 @@ def initialize_kafka_offsets_topic():
         return
     topic = create_random_topic(1, 1)
     produce_example_msg(topic, num_messages=1)
-    create_consumer_group(topic, 'foo')
-    call_offset_get('foo', storage='kafka')
+    kafka = KafkaToolClient(KAFKA_URL)
+    set_consumer_offsets(kafka, "group", {topic: {0: 0}}, offset_storage='kafka')
```

Comment:
I'm pretty sure at some point in my experiments i started generating the group id randomly. I could probably go back to that solution.